<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GammaRay::QuickInspectorWidget</class>
 <widget class="QWidget" name="GammaRay::QuickInspectorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6">
   <item>
    <widget class="QComboBox" name="windowComboBox"/>
   </item>
   <item>
    <widget class="QSplitter" name="splitter_6">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QSplitter" name="previewTreeSplitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QTabWidget" name="tabWidget">
       <property name="tabPosition">
        <enum>QTabWidget::South</enum>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="itemsTab">
        <attribute name="title">
         <string>Items</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="KFilterProxySearchLine" name="itemTreeSearchLine"/>
         </item>
         <item>
          <widget class="QTreeView" name="itemTreeView">
           <attribute name="headerShowSortIndicator" stdset="0">
            <bool>true</bool>
           </attribute>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="sceneGraphTab">
        <attribute name="title">
         <string>Scene Graph</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="KFilterProxySearchLine" name="sgTreeSearchLine"/>
         </item>
         <item>
          <widget class="QTreeView" name="sgTreeView">
           <attribute name="headerShowSortIndicator" stdset="0">
            <bool>true</bool>
           </attribute>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
     <widget class="QStackedWidget" name="stackedWidget">
      <widget class="QWidget" name="page_13">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="GammaRay::PropertyWidget" name="itemPropertyWidget" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_14">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="GammaRay::PropertyWidget" name="sgPropertyWidget" native="true"/>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KFilterProxySearchLine</class>
   <extends>QWidget</extends>
   <header>kde/kfilterproxysearchline.h</header>
  </customwidget>
  <customwidget>
   <class>GammaRay::PropertyWidget</class>
   <extends>QWidget</extends>
   <header location="global">ui/propertywidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>tabWidget</sender>
   <signal>currentChanged(int)</signal>
   <receiver>stackedWidget</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>188</x>
     <y>122</y>
    </hint>
    <hint type="destinationlabel">
     <x>381</x>
     <y>163</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
